modules/openapi-generator/src/main/resources/go/api.mustache

```diff
@@ -342,9 +342,9 @@ func (a *{{{classname}}}Service) {{nickname}}Execute(r {{#structPrefix}}{{&class
 		return {{#returnType}}localVarReturnValue, {{/returnType}}localVarHTTPResponse, err
 	}
 
-	localVarBody, err := ioutil.ReadAll(localVarHTTPResponse.Body)
+	localVarBody, err := io.ReadAll(localVarHTTPResponse.Body)
 	localVarHTTPResponse.Body.Close()
```

Comment:
could the `err` returned by `close()` be either captured and returned or ignored? This will remove some linting warnings

Reply:
@acabarbaye can you please file a PR separately for that?